packages/app/src/messages/reducers/messages.reducer.ts

```diff
@@ -352,5 +358,25 @@ export const messagesReducer = createReducer<MessagesState>(
           }
         }
       )
+      .addCase(
+        fulfilledAction(MessagesEvent.SelectAll),
+        (state, action: SelectAllItemsAction) => {
+          return {
+            ...state,
+            selectedItems: { rows: action.payload },
+          }
+        }
+      )
+      .addCase(MessagesEvent.ResetItems, (state) => {
+        return { ...state, selectedItems: { rows: [] } }
+      })
+      .addCase(MessagesEvent.ToggleItem, (state, action: ToggleItemAction) => {
+        const id = action.payload
+        const threadIds = toggleItemSelect(state.selectedItems, id)
```

Comment:
Please move this logic to the `action`

Reply:
Moved in [106999b](https://github.com/mudita/mudita-center/pull/864/commits/106999bfe9a7c55f9033a87e93120de5f3ffffd0)